engine/gui/src/gui.cpp

```diff
@@ -2017,7 +2017,7 @@ Result DeleteDynamicTexture(HScene scene, const dmhash_t texture_hash)
 
             {
                 uint32_t profiler_hash = 0;
-                const char* profiler_string = dmScript::GetProfilerString(L, custom_ref != LUA_NOREF ? -5 : 0, scene->m_Script->m_SourceFileName, SCRIPT_FUNCTION_NAMES[SCRIPT_FUNCTION_ONMESSAGE], message_name, &profiler_hash);
+                const char* profiler_string = dmScript::GetProfilerString(L, custom_ref != LUA_NOREF ? -5 : 0, scene->m_Script->m_SourceFileName, SCRIPT_FUNCTION_NAMES[script_function], message_name, &profiler_hash);
```

Comment:
it's not necessary on_message function. Pay attention to this switch: https://github.com/defold/defold/blob/fb03e552143db857077aad8d9dce67db927845a2/engine/gui/src/gui.cpp#L1795